infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -120,6 +121,13 @@ public void dispatchHandler(IFailureHandlerPolicy failureHandlerPolicy, Layout c
             if (corfuRuntime.getLayoutView().getLayout().equals(newLayout)) {
                 log.info("Failed node removed. New Layout committed = {}", newLayout);
             }
+
+            //TODO: Since sequencer reset is moved after paxos. Make sure the runtime has the latest
+            //TODO: layout view and latest client router epoch. (Use quorum layout fetch.)
+            //TODO: Handle condition if primary sequencer is not marked ready, reset fails.
+            // Reconfigure servers if required
+            reconfigureServers(corfuRuntime, currentLayout, newLayout, false);
```

Comment:
I can address this in another PR as it might take time to make sure its correct.